x-pack/legacy/plugins/cross_cluster_replication/index.js

```diff
@@ -15,7 +15,7 @@ export function crossClusterReplication(kibana) {
     id: PLUGIN.ID,
     configPrefix: 'xpack.ccr',
     publicDir: resolve(__dirname, 'public'),
-    require: ['kibana', 'elasticsearch', 'xpack_main', 'remote_clusters', 'index_management'],
+    require: ['kibana', 'elasticsearch', 'xpack_main', 'remoteClusters', 'index_management'],
```

Comment:
@joshdover Do you know how this `require` config option works?

Reply:
Ah just saw this. In legacy it causes Kibana to crash if an enabled plugin depends on a plugin that is not enabled. I believe it only looks at other legacy plugins.